core/src/test/java/org/apache/calcite/rel/RelCollationTest.java

```diff
@@ -84,18 +84,36 @@
         is(true));
   }
 
-  /** Unit test for {@link RelCollations#containsOrderless(List, List)}. */
+  /** Unit test for {@link RelCollations#collationsContainKeysOrderless(List, List)}. */
```

Comment:
maybe we could also add some unit tests for the new method `keysContainCollationsOrderless`?

Reply:
Makes sense. Added the test.